lib/html_parser.dart

```diff
@@ -440,11 +440,13 @@ class HtmlParser {
             crossAxisAlignment: CrossAxisAlignment.start,
           );
         case "p":
-          return RichText(
-              text: TextSpan(
-            children: _parseInlineElement(node),
-            style: defaultTextStyle,
-          ));
+          return Padding(
+            padding: EdgeInsets.only(bottom: 10.0),
```

Comment:
Could you set the padding to 1 em (14.0) on both the top and bottom to match other block elements?